decoder/decoder.go

```diff
@@ -13,18 +13,41 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/elastic/apm-server/utility"
+	"github.com/elastic/beats/libbeat/monitoring"
 )
 
 type Reader func(req *http.Request) (io.ReadCloser, error)
 type Decoder func(req *http.Request) (map[string]interface{}, error)
 
+var (
+	decoderMetrics        = monitoring.Default.NewRegistry("apm-server.decoder", monitoring.PublishExpvar)
+	deflateLengthAcc      = monitoring.NewInt(decoderMetrics, "deflate.content-length")
+	gzipLengthAcc         = monitoring.NewInt(decoderMetrics, "gzip.content-length")
+	uncompressedLengthAcc = monitoring.NewInt(decoderMetrics, "uncompressed.content-length")
+	readerAcc             = monitoring.NewInt(decoderMetrics, "reader.size")
```

Comment:
What does `Acc` stand for here?

Reply:
accumulator. i am not counting with `Inc()`, but accumulating up large numbers, i want to make explicit how these monitors are used.